WordPress/Docs/PHP/PregQuoteDelimiterStandard.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+               title="Preg Quote Delimiter"
+>
+    <standard>
+        <![CDATA[
+    Passing the $delimiter parameter to preg_quote() is strongly recommended to ensure the delimiter required by the PCRE functions are escaped.
```

Comment:
> to ensure the delimiter required by the PCRE functions are escaped.

I haven't looked deeply into this so I might be missing something, but I'm not sure this is the reason to recommend that the `$delimiter` parameter is passed.

I got this from the [issue where this sniff was introduced](https://github.com/WordPress/WordPress-Coding-Standards/issues/1371):

> Verify that the optional second parameter $delimiter of preg_quote() is always passed.
> Too often I come across code where it is missing and unless it is passed, the default / delimiters are presumed, which is often wrong.

Reply:
Updated the description and changed the example valid code to be one from WordPress core.